.travis.yml

```diff
@@ -37,6 +37,7 @@ addons:
       - gcc
 
 install:
+    - pip install git+https://git@github.com/equinor/stea.git
```

Comment:
It is possible to add a repository directly to the requirements.txt file, please do that instead.

i.e. `git+git://github.com/equinor/stea` should make it possible to install stea via `pip install -r requirements.txt`

Reply:
will do!